db/migrate/20190915141533_create_videos.rb

```diff
@@ -0,0 +1,10 @@
+class CreateVideos < ActiveRecord::Migration[6.0]
+  def change
+    create_table :videos do |t|
+      t.references :run, index: true
+      t.string :url
```

Comment:
Do you think these fields should be non-nullable?

Reply:
Probably. What is the point of a `Video` without a `URL`.